composer/algorithms/ghost_batchnorm/ghost_batchnorm.py

```diff
@@ -161,30 +159,42 @@ def _has_momentum(self) -> bool:
 
     def forward(self, input: torch.Tensor) -> torch.Tensor:  # type: ignore
         batch_size = input.shape[0]
+
         if batch_size < self.ghost_batch_size:
             raise ValueError(f"Worker batch size {batch_size} < ghost_batch_size {self.ghost_batch_size}")
 
-        nchunks = int(np.ceil(batch_size / self.ghost_batch_size))
-        has_momentum = self._has_momentum()
-        if has_momentum:
+        nchunks: int = int(math.ceil(batch_size / self.ghost_batch_size))
+        scale = self.batchnorm.momentum is not None
```

Comment:
Nit: any reason to avoid calling this `has_momentum`? `scale` is ambiguous to me